src/containers/ProjectCard/ProjectCard.js

```diff
@@ -10,30 +11,39 @@ class ProjectCard extends Component {
     this.props.setActivePalette(palette)
   }
 
-  getPalettes = (e) => {
-    e.preventDefault()
+  deleteProject = () => {
+    const { id } = this.props.project
+    const project = { id }
+    const url = process.env.REACT_APP_BACKEND_URL + `/api/v1/projects/${id}`
+    this.props.handleProject(url, deleteProjectSuccess, 'DELETE', project)
+  }
+
+  deletePalette = (pID) => {
+    const projID = this.props.project.id
+    const palette = { pID }
+    const url = process.env.REACT_APP_BACKEND_URL + `/api/v1/projects/${projID}/palettes/${pID}`
```

Comment:
I like the destructuring above to make the URL interpolation a little easier to digest